hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketArgs.java

```diff
@@ -187,6 +187,27 @@ public Map<String, String> toAuditMap() {
     if (this.ownerName != null) {
       auditMap.put(OzoneConsts.OWNER, this.ownerName);
     }
+    if (this.quotaInBytesSet && quotaInBytes > 0 ||
+        !(this.quotaInBytes == OzoneConsts.QUOTA_RESET)) {
```

Comment:
nit: can be simplified?

```suggestion
        (this.quotaInBytes != OzoneConsts.QUOTA_RESET)) {
```